fiftyone/utils/paddleocr.py

```
@@ -0,0 +1,418 @@
+                if crop is not None and crop.size > 0:
+                    crops.append(crop)
+                    crop_idx.append(i)
+
+            if crops:
+                try:
+                    rec_out = list(
+                        self._rec_model.predict(
+                            input=crops, batch_size=len(crops)
+                        )
+                    )
+                    for i, res in zip(crop_idx, rec_out):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)